### DISCUSSION TITLE
FQDN for jgroup communication

### DISCUSSION THREAD

**Discussion by amolb1** [belaban/JGroups#658] [Q&A]
I am naive to Jgroups and wanted to know does jgroup support FQDN's instead of IP address for communication ?.

**Comment by belaban** [comment]
To what purpose? Run it over Rest-like endpoints?

No, this is not supported. To support it, you'd have to (a) create a 
Rest-based transport and (b) use custom address generation [1].
Cheers


[1] http://www.jgroups.org/manual5/index.html#CustomAddresses

On 12.09.22 13:25, amolb1 wrote:
> I am naive to Jgroups and wanted to know does jgroup support FQDN's 
> instead of IP address for communication ?.
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/belaban/JGroups/discussions/658>, or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AADPZXNW35NWBO3JEHSZUXTV54HIZANCNFSM6AAAAAAQKLZINE>.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 

-- 
Bela Ban | http://www.jgroups.org

**Comment by rhusar** [comment]
If the question is solely about using FQDN in e.g. `TCPPING`'s `initial_hosts` instead of IP addresses that's perfectly fine.
